docs/benefits_of_contextualized_ML.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>Remove the mention of gene expression. We can add use-case examples in the introduction but the examples should focus on the more abstract notion of context-dependent networks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>